### ISSUE TITLE
UUID issue with either Deep or ElecCore

### ISSUE THREAD

**Issue by whatthedrunk** [McJty/DeepResonance#162]
I tried adding the following mods to ATM2 but when you login you seem to have a have a random UUID and have the default Alex skin. At first I thought thought my player data was corrupted but it also happened on a fresh install.  

deepresonance-1.1x-1.4.2
ElecCore-1.11-1.6.377
Tested with Forge 2206 and a 2203

**Comment by McJty** [comment]
@Elec332 any idea what this could be?

**Comment by McJty** [comment]
Does it go away if you remove deepresonance but keep eleccore?
What if you remove both mods?

Also logs would be helpful

**Comment by whatthedrunk** [comment]
It works fine in single player but not on servers. Let me make a profile with just those mods really quick to pinpoint it.

**Comment by whatthedrunk** [comment]
http://pastebin.com/G1UAiXbc fml-server log 
It seem's to be broken with just eleccore and only on servers so feel free to close this and post the issue there or you can direct me to where I can submit it. Let me know if you need any other logs.

**Comment by McJty** [comment]
The UUID of the player is something that neither eleccore nor deep resonance touches. Maybe there was an issue with the mojang services at that time or something. But this is not something mods can control really

**Comment by Elec332** [comment]
The MC skin services aren't reallt known for their stability either, won't be the first time I log into a (vanilla) server and find that my skin doesn't work...

**Comment by whatthedrunk** [comment]
Yeah i'm not sure what to tell you then. Have you tried setting up a local server with with eleccore in it and then again without it? With it out my skin/uuid is fine and when i add it both change this also works for other people trying to login so I can't add either mod to the 1.11.2 packs yet.

**Comment by McJty** [comment]
@Elec332 I can confirm this is a problem. I tried on a local server with only ElecCore and it is *always* wrong (1.11.2 only). On a local server with only Forge and no ElecCore it is always right.
I also noticed that I have a different UUID. So the server contains:

cat usernamecache.json 
{
  "ff65ca82-f788-4308-96ad-ce2e5c513d6f": "McJty",
  "750765da-470c-32b1-965e-2962de70a03f": "McJty"
}


cat usercache.json 
[{"name":"McJty","uuid":"750765da-470c-32b1-965e-2962de70a03f","expiresOn":"2017-02-09 19:52:18 +0100"},{"name":"McJty","uuid":"ff65ca82-f788-4308-96ad-ce2e5c513d6f","expiresOn":"2017-02-09 19:47:24 +0100"}]

The first McJty from the case where I joined without ElecCore
